canonical-websites/www.ubuntu.com
# Added the kubernetes takeover

## Done

Added Improving Productivity with kubernetes takeover
## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Refresh the homepage until you see the "Improving Productivity with Kubernetes" takeover shows up Ensure it matches the [copy doc](https://docs.google.com/document/d/16YVNq8GST-yD7B3jhSwQ7_aZ0fc7Ly7l5MD4TQAhAXQ/edit?ts=5bd9ac7e) and [design spec](https://github.com/ubuntudesign/web-squad/issues/883)


## Issue / Card

Fixes https://github.com/ubuntudesign/web-squad/issues/884
